docs/reference/commandline/plugin_create.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "plugin create"
```

Comment:
this should be `"plugin create (experimental)"`

in the new docs rendering, this is actually used to create the H1 heading